Tests/MasseTests/MP3DurationTests.swift

```diff
@@ -0,0 +1,80 @@
+import XCTest
+@testable import Masse
+
+final class MP3DurationTests: XCTestCase {
+    var paths: [String: String] = [
+        // Generated with `ffmpeg -i filename`
+        "Tests/MasseTests/Resources/ID3v1.mp3": "00:06:38",
+        "Tests/MasseTests/Resources/Truncated.mp3": "00:03:26",
+        "Tests/MasseTests/Resources/VBR9.mp3": "00:06:38"
+    ]
+    override func setUp() {
+        let filemanager = FileManager.default
+        let folder = filemanager.currentDirectoryPath
+        var newPaths: [String: String] = [:]
+        // Preflight-Check to make sure the mp3s exist
+        for path in paths.keys {
+            let fullPath = "\(folder)/\(path)"
+            guard filemanager.fileExists(atPath: fullPath) else {
+                longAwfulErrorMessage()
+                return
+            }
+            newPaths[fullPath] = paths[path]
+        }
+        paths = newPaths
+    }
+    
+    func testMp3Durations() {
+        for (path, staticDuration) in paths {
+            let url = URL(fileURLWithPath: path)
+            do {
+                let mp3Duration = try MP3DurationCalculator(url: url)
```

Comment:
[`XCTAssertNoThrow`](https://developer.apple.com/documentation/xctest/2852885-xctassertnothrow) might give better failure outputs than the manual `catch`.